qemu/tests/cfg/virtiofs_supp_for_fs.cfg

```diff
@@ -0,0 +1,55 @@
+- virtiofs_supp_for_fs:
+    type = virtiofs_supp_for_fs
+    no Linux
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    kill_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = ${mem}M
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    io_timeout = 600
+    fs_dest = '/mnt/${fs_target}'
+    guest_username = "user00001"
+    guest_username_pwd = "U1pwd345?a"
+    add_user_cmd = "net user ${guest_username} ${guest_username_pwd} /add"
+    del_user_cmd = "net user ${guest_username} /delete"
+    fs_source_dir = virtio_fs_test/
+    driver_name = viofs
+    virtio_win_media_type = iso
+    cdroms += " virtio"
+    # install winfsp tool
+    i386, i686:
+        install_winfsp_path = 'C:\Program Files'
+    x86_64:
+        install_winfsp_path = 'C:\Program Files (x86)'
+    s390, s390x:
```

Comment:
I think multi-arch not support windows guest now. cc @zhenyzha could you confirm it ?

Reply:
> @zhenyzha

Hello @zhenyzha , can you give us some comments? 
Thanks!

Reply:
Yes, the current arm and PPC do not cover the test scenarios of Windows. 
I believe the same case with the S390. 
Feel free to correct me if I'm mistaken. @fbq815

Reply:
zhenyzha is right, we don't support windows on s390x as well